src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -0,0 +1,308 @@
+    grant_request_2 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_1.json')))
+    grant_request_3 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_2.json')))
+    grant_request_4 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_3.json')))
+    grant_request_5 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_request_6 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_1.json')))
+    grant_request_7 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_2.json')))
+    grant_request_8 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_3.json')))
+
+    # device_b device_d and device_h are of Category B
+    # Load Conditional Data
+    self.assertEqual(device_2['cbsdCategory'], 'B')
+    conditionals_device_2 = {
+        'cbsdCategory': device_2['cbsdCategory'],
+        'fccId': device_2['fccId'],
+        'cbsdSerialNumber': device_2['cbsdSerialNumber'],
+        'airInterface': device_2['airInterface'],
+        'installationParam': device_2['installationParam'],
+        'measCapability': device_2['measCapability']
+    }
+    self.assertEqual(device_4['cbsdCategory'], 'B')
+    conditionals_device_4 = {
+        'cbsdCategory': device_4['cbsdCategory'],
+        'fccId': device_4['fccId'],
+        'cbsdSerialNumber': device_4['cbsdSerialNumber'],
+        'airInterface': device_4['airInterface'],
+        'installationParam': device_4['installationParam'],
+        'measCapability': device_4['measCapability']
+    }
+    self.assertEqual(device_8['cbsdCategory'], 'B')
+    conditionals_device_8 = {
+        'cbsdCategory': device_8['cbsdCategory'],
+        'fccId': device_8['fccId'],
+        'cbsdSerialNumber': device_8['cbsdSerialNumber'],
+        'airInterface': device_8['airInterface'],
+        'installationParam': device_8['installationParam'],
+        'measCapability': device_8['measCapability']
+    }
+
+    conditionals = {'registrationData': [conditionals_device_2, conditionals_device_4, conditionals_device_8]}
+    # Remove conditionals from registration
+    del device_2['cbsdCategory']
+    del device_2['airInterface']
+    del device_2['installationParam']
+    del device_2['measCapability']
+    del device_4['cbsdCategory']
+    del device_4['airInterface']
+    del device_4['installationParam']
+    del device_4['measCapability']
+    del device_8['cbsdCategory']
+    del device_8['airInterface']
+    del device_8['installationParam']
+    del device_8['measCapability']
+
+    # Load GWPZ Record
+    gwpz_record_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'gwpz_record_0.json')))
+
+    # Load FSS record
+    fss_record_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'fss_record_0.json')))
+
+    # Load ESC record
+    esc_record_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'esc_sensor_record_0.json')))
+
```

Comment:
esc_sensor_record_0.json is incomplete...missing protectionLevel (can be fixed outside this PR)

Reply:
protectionLevel is optional, isn't it?

Reply:
I guess it is......it would be so much simpler if things were just required or not..................

The protection level to be applied to this ESC sensor in units of dBm/MHz. If not present, the level should be interpreted as equal to that indicated in (n.8, R2-ESC-07)

Reply:
Yes, protectionLevel is optional. We may need to discuss if we need to add it in for the test case.